changelog/unreleased/pr-14424.toml

```diff
@@ -17,8 +17,8 @@ with a minimum of user input. The default settings are based on ES/OS sharding r
 do not need to be changed.
 The following configuration file settings are available, if you wish to modify the behavior
 for your requirements:
-`time_size_optimizing_rotation_min_shard_size`: index should be at least this big before rotating.
-`time_size_optimizing_rotation_max_shard_size`: rotate if index exceeds this size limit.
-`time_size_optimizing_rotation_min_lifetime`: index should be at least this old before rotating.
-`time_size_optimizing_rotation_max_lifetime`: rotate if index exceeds this age limit.
+`time_size_optimizing_rotation_min_shard_size=20g`: shard should be at least this big before rotating.
+`time_size_optimizing_rotation_max_shard_size=50g`: rotate if shard exceeds this size limit.
+`time_size_optimizing_retention_min_lifetime=30d`: index should be at least this old before rotating.
+`time_size_optimizing_retention_max_lifetime=40d`: rotate if index exceeds this age limit.
```

Comment:
The descriptions do not match the new config option name (retention vs. rotation).

Would the following suggestion describe the behaviour correctly?

```suggestion
`time_size_optimizing_retention_min_lifetime=30d`: minimum index age for retention configuration to be applied.
`time_size_optimizing_retention_max_lifetime=40d`: maximum index age for retention configuration to be applied.
```

Reply:
I think the description in the UI is actually very good:

<img width="1035" alt="Screenshot 2023-02-21 at 10 09 42" src="https://user-images.githubusercontent.com/4270/220301882-c8e942cd-b0f6-4797-90e1-5f11d9d8ffff.png">

Maybe we should just use the same wording here.